client/app/cloud/project/compute/infrastructure/openstackClient/cloud-project-compute-infrastructure-openstack-client.html

```diff
@@ -1,4 +1,6 @@
-<div id="openstackClient" data-ng-controller="CloudProjectComputeInfrastructureOpenstackClientCtrl as OpenstackClientCtl" >
+<div id="openstackClient"
+     data-ng-controller="CloudProjectComputeInfrastructureOpenstackClientCtrl as OpenstackClientCtl"
+     data-ng-init="OpenstackClientCtl.initWithConfig(OpenstackClientCtl.$stateParams.hTerm)">
```

Comment:
Can be done in $onInit to prevent people from having to look in the HTML to discover this behavior.